src/ptf/testutils.py

```diff
@@ -859,7 +859,8 @@ def simple_gre_packet(pktlen=300,
 
     if inner_frame:
         pkt = pkt / inner_frame
-        if ((ord(str(inner_frame)[0]) & 0xF0) == 0x60):
+        inner_frame_bytes = bytearray(bytes(inner_fram))
```

Comment:
typo: `inner_frame`.
Also, AFAICT, you can directly construct a bytearray, without first converting to bytes: https://docs.python.org/3.7/library/stdtypes.html?highlight=bytearray#bytearray

Reply:
I tried it with just bytearray() directly but if inner_frame is a scapy pkt object, then it won't work. So converting it to bytes first is a safe way of converting to a common representation.